### DISCUSSION TITLE
The hairline is nearly invisible

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#43965] [Mobile] [answered]
I loaded a PR and was trying to see the latest comment, there's technically a hairline showing me that the comments button isn't the comments button I'm looking for, but on this fairly high dpi display, it's nearly invisible and I didn't recognize that the comments button was for a pair of review comments:

![Screenshot_20230111-093912.png](https://user-images.githubusercontent.com/2119212/211834699-d0fd4caf-2ed2-4e81-81cd-672a94816383.png)
![Screenshot_20230111-093851.png](https://user-images.githubusercontent.com/2119212/211834720-6d7cc2b1-0234-47bf-be63-5323b57e41a5.png)

I'd suggest the line be made thicker/darker. 

Aliso, I don't think that using text for the overlay button is valuable, a 💬 speech bubble should work nicely, won't suffer from being even wider when translated, and will make it easier to read the view.

**Comment by HenSquared** [comment, marked as answer]
Thanks! I've pinged designers this feedback to get their thoughts!
